### DISCUSSION TITLE
无法下载？无法抓取？

### DISCUSSION THREAD

**Discussion by xifangczy** [xifangczy/cat-catch#28] [General]
猫抓是通用的扩展，扩展的权限很低能做的事情很少，也不会为单独某个网站编写代码。

推荐几个比较好用的工具。

youtube-dl 通用嗅探下载，不只是Youtube
https://github.com/ytdl-org/youtube-dl

yt-dlp 通用嗅探下载，youtube-dl的继承者，更新更频繁。
https://github.com/yt-dlp/yt-dlp

you-get 通用嗅探下载，老牌应用
https://github.com/soimort/you-get

通用嗅探下载 正版需要付费
https://www.internetdownloadmanager.com/

Bilibili专用下载扩展
https://chrome.google.com/webstore/detail/kpbnombpnpcffllnianjibmpadjolanh

更多好工具，欢迎提交讨论。

**Comment by StrollStars** [comment]
CLI：[Lux](https://github.com/iawia002/lux)、[ykdl](https://github.com/SeaHOH/ykdl)、[you-get](https://you-get.org/)、[yt-dlp](https://github.com/yt-dlp/yt-dlp)
GUI：[youtube-dl-gui](https://jely2002.github.io/youtube-dl-gui/)、[media-downloader](https://github.com/mhogomchungu/media-downloader)

**Reply by kgduhao** [reply]
无法获取视频地址，缓存模式、录屏模式全部无法捕获数据，该如何解决？
